src/sentry/tasks/symbolication.py

```diff
@@ -161,6 +162,7 @@ def _do_symbolicate_event(
 
     # Backwards compatibility: If the current platform is JS, we may need to do
     # native afterwards. Otherwise we don't do anything.
+    # TODO @athena: remove comment before merge
```

Comment:
According to the comment above, nothing here needs to change. We won't need this for new events.